index.js

```diff
@@ -14,14 +14,13 @@ const flip = async (currentPath) => {
     // Read config file content
     let configContent = await readConf(currentPath);
 
-    const httpsPathRegex = new RegExp(GH_PATH.HTTPS_PATH, 'i');
+    const httpsPathRegex = new RegExp('https?://github.com/', 'i');
```

Comment:
I think variable name `httpPathRegex` will be better.

This looks like a new constant.
Let's move it to the [gh-conf-constants](https://github.com/gh-conf/gh-conf-constants) repository.
I have created an issue for this here [gh-conf/gh-conf-constants#3](https://github.com/gh-conf/gh-conf-constants/issues/3)

Would you like to work on it?

Reply:
I didn't know that it is the library(gh-conf-constants) you are participating in 😯

Yes! I'll do it!